_gsocprojects/2020/project_CMS.md

```diff
@@ -0,0 +1,10 @@
+---
+project: CMS
+layout: default
+logo: CMS-Color.gif
+description: |
+  [CMS](http://cms.cern/) is a high-energy physics experiment at the LHC.
```

Comment:
I would put a more verbose description here, up to 50 words. At least mention in few words what CMS is searching for.